visual-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "playwright",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "node ./scripts/visual-tests-build.mjs",
+    "docker:run": "docker pull mcr.microsoft.com/playwright:v1.28.1-focal && docker run -v $(INIT_CWD)/..:/tests/ -w /tests/visual-test --rm -it mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash",
+    "docker:run:win": "docker pull mcr.microsoft.com/playwright:v1.28.1-focal && docker run -v %INIT_CWD%/..:/tests/ -w /tests/visual-test --rm -it mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash",
+    "upload": "node ./scripts/visual-tests-upload.mjs",
+    "all": "node ./scripts/visual-tests-build.mjs && node ./scripts/visual-tests-upload.mjs"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Just checking - is the license entry added here on purpose, or is it a leftover of the `package.json` auto-generating script?

Reply:
it's leftover